charts/redpanda/templates/secrets.yaml

```diff
@@ -14,6 +14,12 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */}}
+{{- $values := .Values -}}
+{{- $internalAdvertiseAddress := printf "%s.%s" "${SERVICE_NAME}" (include "redpanda.internal.domain" .) -}}
+{{- $externalAdvertiseAddress := printf "${SERVICE_NAME}" -}}
+{{- if (tpl ($values.external.domain | default "") $) -}}
+  {{- $externalAdvertiseAddress = printf "${SERVICE_NAME}.%s" (tpl $values.external.domain $) -}}
+{{- end -}}
```

Comment:
This comes from statefulset.yaml, so why it was not removed from there?

Reply:
GOOD POINT! I wasnt sure if it was being used for something else, ill check and remove if nothing else is using it.